examples/tests/invalid/impl_interface.w

```diff
@@ -17,3 +17,19 @@ resource C impl cloud.Bucket {
               // ^^^^^^^^^^^ Error: cloud.Bucket is a resource, not an interface
   init() {}
 }
+
+interface I1 {
+  method_1(x: num): num;
+}
+
+interface I2 extends I1 {
+  inflight method_2(x: str): str;
+}
+
+interface I3 extends I2 {
+  method_3(x: Array<num>): Array<num>;
+}
+
+resource r impl I3 {
+  init() {}
```

Comment:
Can we add a comment saying what the error here is?

Reply:
Yes - good one. Added